packages/ckeditor5-link/src/linkui.ts

```diff
@@ -556,6 +552,30 @@ export default class LinkUI extends Plugin {
 
 			return button;
 		} );
+
+		editor.ui.componentFactory.add( 'linkProperties', locale => {
+			const linkCommand: LinkCommand = editor.commands.get( 'link' )!;
+			const button = new ButtonView( locale );
+			const t = locale.t;
+
+			button.set( {
+				label: t( 'Link properties' ),
+				icon: icons.settings,
+				tooltip: true
+			} );
+
+			button.bind( 'isEnabled' ).to(
+				linkCommand, 'isEnabled',
+				linkCommand, 'manualDecorators',
+				( isEnabled, manualDecorators ) => isEnabled && manualDecorators.length > 0
+			);
```

Comment:
As this component could be added to the main toolbar (not only to the link toolbar) it should also be disabled when the link command value is empty - the selection is not on the link.